mlrun/projects/pipelines.py

```diff
@@ -721,165 +728,69 @@ class _RemoteRunner(_PipelineRunner):
-            artifact_path=artifact_path,
-            workflow_handler=workflow_handler,
-            namespace=namespace,
-        )
-
-        if workflow_spec.schedule:
-            is_scheduled = True
-            schedule_name = runspec.spec.parameters.get("workflow_name")
-            run_db = mlrun.get_run_db()
-
-            try:
-                run_db.get_schedule(project.name, schedule_name)
-            except mlrun.errors.MLRunNotFoundError:
-                is_scheduled = False
-
-            if workflow_spec.override:
-                if is_scheduled:
-                    logger.info(f"Deleting schedule {schedule_name}")
-                    run_db.delete_schedule(project.name, schedule_name)
-                else:
-                    logger.info(
-                        f"No schedule by name '{schedule_name}' was found, nothing to override."
-                    )
-            elif is_scheduled:
-                raise mlrun.errors.MLRunConflictError(
-                    f"There is already a schedule for workflow {schedule_name}."
-                    " If you want to override this schedule use 'override=True' (SDK) or '--override-workflow' (CLI)"
-                )
-
-        # The returned engine for this runner is the engine of the workflow.
-        # In this way wait_for_completion/get_run_status would be executed by the correct pipeline runner.
-        inner_engine = get_workflow_engine(workflow_spec.engine)
-
-        msg = "executing workflow"
-        if workflow_spec.schedule:
-            msg += " scheduling"
+        workflow_action = "schedule" if workflow_spec.schedule else "run"
         logger.info(
-            f"{msg} '{load_and_run_fn.metadata.name}' remotely with {workflow_spec.engine} engine"
+            f"submitting {workflow_action} request for '{workflow_name}' workflow"
         )
 
+        workflow_id = None
+        inner_engine = get_workflow_engine(workflow_spec.engine)
+        run_db = mlrun.get_run_db()
         try:
-            run = load_and_run_fn.run(
-                runspec=runspec,
-                local=False,
-                schedule=workflow_spec.schedule,
+            workflow_response = run_db.submit_workflow(
+                project=project.name,
+                name=workflow_name,
+                workflow_spec=workflow_spec,
                 artifact_path=artifact_path,
+                source=source,
+                run_name=mlrun.mlconf.default_workflow_runner_name.format(
+                    workflow_name
+                ),
+                namespace=namespace,
             )
             if workflow_spec.schedule:
                 return
-            # Fetching workflow id:
-            while not run_id:
-                run.refresh()
-                run_id = run.status.results.get("workflow_id", None)
-                time.sleep(1)
+
+            # Getting workflow id from run:
+            response = retry_until_successful(
+                10,
```

Comment:
Why are you starting with 10 seconds? It seems a bit high - in previous code iteration you started with 1 sec.